tests/test_model.py

```diff
@@ -1062,6 +1062,16 @@ def test_deterministic():
     assert model["y"] == y
 
 
+def test_potential_with_dims():
+    """
+    Test to check the passing of dims to the potential
+    """
+    with pm.Model(coords={"observed": range(10)}) as model:
+        x = pm.Normal("x", 0, 1)
+        y = pm.Deterministic("y", x**2, dims=("observed",))
```

Comment:
This test does not use `Potential`

Reply:
Sorry for this,

I found that dims test case was not available for deterministic as well so added that test case with potential.

I might forget to update the test case for potential.

I have updated the code.

Please review it.